features/step_definitions/authorization.js

```diff
@@ -44,7 +44,7 @@ module.exports = function server() {
         });
     });
 
-    this.Then(/^they can see the project$/, () =>
+    this.Then(/^they can see the project$/, { timeout: TIMEOUT }, () =>
```

Comment:
In the first run it may time out after 5 seconds by default. I want to extend the timeout time.